systemtest/src/test/java/io/strimzi/systemtest/mirrormaker/MirrorMaker2ST.java

```diff
@@ -650,11 +650,11 @@ void testMirrorMaker2CorrectlyMirrorsHeaders() {
         //deploying example clients for checking if mm2 will mirror messages with headers
 
         KafkaBasicClientResource targetKafkaClientsJob = new KafkaBasicClientResource("", targetConsumerName,
-            KafkaResources.plainBootstrapAddress(kafkaClusterTargetName), targetExampleTopic, MESSAGE_COUNT, "", ClientUtils.generateRandomConsumerGroup(), 0);
+            KafkaResources.plainBootstrapAddress(kafkaClusterTargetName), targetExampleTopic, MESSAGE_COUNT, "", ClientUtils.generateRandomConsumerGroup(), 1000);
```

Comment:
Did you update the message count for this test? I just want to avoid waiting there 500 seconds for example instead of 5 seconds.